Doc/reference/simple_stmts.rst

```diff
@@ -664,8 +664,7 @@ and information about handling exceptions is in section :ref:`try`.
 .. versionchanged:: 3.3
     :const:`None` is now permitted as ``Y`` in ``raise X from Y``.
 
-.. versionadded:: 3.3
```

Comment:
similar to my comment in stdtypes.rst - shouldn't we maintain separate entries for added vs changes at the same version?